docker/docker-compose-e2e.yml

```diff
@@ -0,0 +1,35 @@
+# This file is only meant to be run as an override to docker-compose.yml
```

Comment:
This is really interesting-- you can extend compose files with others and they'll smartly merge. So we get the benefit of all 3 normal services (db, server, client) and can add an e2e service, also.